src/wp-includes/blocks.php

```diff
@@ -227,6 +227,15 @@ function get_block_metadata_i18n_schema() {
  * @return WP_Block_Type|false The registered block type on success, or false on failure.
  */
 function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
+	// Get an array of metadata from a PHP file.
```

Comment:
Let's use the [multi-line comment format](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments) here, per the documentation standards.

Reply:
Good catch. Done.